src/BadEndPage.js

```diff
@@ -0,0 +1,33 @@
+import firebase from './firebase.js';
+import { getDatabase, ref, push } from 'firebase/database';
+import UserStatsPage from "./UserStatsPage.js";
+
+function BadEndPage({setDisplay, username, sword, userStats}) {
+    
+    const addUser = (userStats) => {
+        const database = getDatabase(firebase);
+        const dbRef = ref(database);
+        push(dbRef, userStats);
+    }
+
+    return(
+        <div className='displayContainer BadEndPageContainer'>
+            <div className='contentContainer'>
+                <h1>Well, <span className="username">{username}</span></h1>
+                <p>Looks like this is the end...You made some pretty interesting decisions along the way, but don't worry! If you didn't like the way things panned out I encourage you to try again! Afterall, adventurers never give up!</p>
```

Comment:
could this be indented to be inline with above <p> tag or it could also just be the way it looks on github